.github/workflows/build.yml

```diff
@@ -135,9 +138,14 @@ jobs:
           prefix: ${{ github.workspace }}
 
       - name: overlay this pantry
-        run: cp -Rv $GITHUB_WORKSPACE/pantry/ /opt/tea.xyz/var/pantry || true
-        # ^^ || true because I can’t figure out how to stop `cp` erroring
-        # because it doesn’t want to copy symlinks that already exist, eg. `src`
+        run: |
+          tea --sync
```

Comment:
why the `--sync` tho

Reply:
are you expecting gnu cp to be run?

Because it won’t without doing `tea +gnu.org/coreutils cp` instead of `cp`

Reply:
`tea --sync` is unrequired in these contexts. Currently we run it automatically, but I intend to stop that shortly actually and require it to be run manually, then have the installer do it, so for GHA usage you won't have to either.

Reply:
This was to ensure the paths exist. I could use mkdir, but this feels like it will be more correct for more situations.

Reply:
Aah ok.

We need a bot that automatically comments “needs comments” to your PRs :P

Reply:
to be clear, `tea --sync` is run by the action, but that is coincidental because `srcroot` is set to a directory that contains a tea-env. So you are correct to add this. Needs comment tho.

Reply:
`this code works, but don't try to understand why; developer is unhelpful.`